test/coverage-test/test/web-worker.test.ts

```diff
@@ -0,0 +1,67 @@
+import { expect } from 'vitest'
+import { formatSummary, isV8Provider, readCoverageMap, runVitest, test } from '../utils'
+
+test('web worker coverage is correct', async () => {
+  await runVitest({
+    setupFiles: ['@vitest/web-worker'],
+    include: ['fixtures/test/web-worker.test.ts'],
+    environment: 'jsdom',
+    coverage: {
+      include: [
+        // Runs in web-worker's runner with custom context -> execution wrapper ~430 chars
+        'fixtures/src/worker.ts',
+
+        // Runs in default runner -> execution wrapper ~185 chars
+        'fixtures/src/worker-wrapper.ts',
+      ],
+      reporter: 'json',
```

Comment:
Do you manually add `html` here when you want to check report? Or is it fine to have `['json', 'html']` here?

I normally runs a specific project and test case when I check html report like

```js
pnpm -C test/coverage-test test -- --project v8 /web-worker.test
```

Reply:
I've been just adding the HTML here manually. Maybe we should add an environment variable that automatically adds it here 🤔 

https://github.com/vitest-dev/vitest/blob/12eaf3ea4369c46ab576e371e729a7cad9dfb647/test/coverage-test/utils.ts#L42-L48

Reply:
We could have the `html` here always but on CI it would useless and maybe slightly slower. Maybe the reporter should be added automatically when running locally.

Reply:
I just wanted to check if you do something different locally. I'm okay either way :+1:

Btw, `vue.test.ts` always generated html, so maybe you forgot to remove? :smile: 

https://github.com/vitest-dev/vitest/blob/c82387d36f7dc0cf8f558b96bf97f782b0b3637a/test/coverage-test/test/vue.test.ts#L9

Reply:
Ah, `vue.test.ts` asserts on generated html files, so it is on purpose.